packages/ui/src/components/Navigation/Navigation.tsx

```diff
@@ -188,8 +203,8 @@ export function Navigation({
                                         target="_blank"
                                         rel="noopener noreferrer"
                                     >
-                                        {new URL(externalSiteLink).hostname}
-                                        <ArrowUpRightIcon className="ml-1 w-2 h-2" />
+                                        {externalLinkLabel}
+                                        <ArrowUpRightIcon className="ml-1 w-4 h-4" />
```

Comment:
@fgyimah I noticed quite some difference between our nav and yours, both in icon size and spacing between them

![CleanShot 2023-11-14 at 17 42 30@2x](https://github.com/prezly/theme-kit-js/assets/858646/289fd86d-47d6-4ec8-bb17-451670577592)

Note we've used the heroicons mini for the dropdowns and external links - they are smaller but bolder, and you chouls be able to use them at fullsize (20px) without having to resize them. Other standalone icons use the fullsize 24px versions, but thought I'd let you know in case you missed that difference!

![CleanShot 2023-11-14 at 17 44 08@2x](https://github.com/prezly/theme-kit-js/assets/858646/1fac4b46-858e-4ef2-9afa-6e708f2e24bd)

Reply:
Updated